events/2021-kubevirt-summit/proposals/kubevirt-integration-in-action.md

```diff
@@ -0,0 +1,25 @@
+# Title
+
+## KubeVirt Integration in Action
+### Developing a Production Ready Gardener Setup
+
+# Abstract
+
+This talk shares an experience of integrating KubeVirt as a cloud provider at Gardener. During our session, we will discuss how we developed the KubeVirt extension plugins used in on-premise environments and how to prepare a production-ready setup by using Multus, DataVolumes and future plans to use the hotplug volume feature.
+
+# Presenters
+
+- Huamin Chen, Senior Principal Software Engineer, Red Hat, hchen@redhat.com, github.com/rootfs
+- Marcin Franczyk, Senior Software Engineer, Kubermatic, marcin0franczyk@gmail.com, github.com/mfranczy
+
+[X] The presenters agree to abide by the
+    [Linux Foundation's Code of Conduct for Events](https://events.linuxfoundation.org/about/code-of-conduct/)
+
+# Session details
+
+- Track: User
+- Session type: Presentation
+- Duration: 40m
```

Comment:
One question: do you think it would be possible to deliver this session within a 20 min slot?

We are working to schedule as many proposals as possible, and it should help if some sessions can be shorter, hence the question.

Reply:
Yes, 20 minutes is ok, we will keep the most significant information. I think 40 minutes would be too long for this session.